junipervpn/tncc.py

```diff
@@ -78,9 +78,9 @@ def decode_0ce4(buf, indent):
 
 # 0ce5 - string without hex prefixer
 def decode_0ce5(buf, indent):
-    s = struct.unpack(str(len(buf)) + "s", buf)
```

Comment:
This all seems OK - although I'm surprised this wasn't one of the first things in the series... does it still work with Python3 without this patch?

(I mean, we should definitely commit it, but it would be cool to ensure any commit within this PR will work).